Profiles - ZIB 2017/vitalsigns-bodyheight.xml

```diff
@@ -35,6 +35,18 @@
       <definition value="Body Height." />
       <comment value="additional codes that translate or map to this code are allowed.  For example a more granular LOINC code or code that is used locally in a system." />
       <alias value="Lichaamslengte" />
+      <binding>

```

Comment:
Since this issue is specifically about mapping HCIM codes to vitalsigns, the ConceptMap only needs to be applied to the HCIM profile, not to this base profile.

Reply:
Removed the binding at the vitalsigns-bodyheight profile.